main.go

```diff
@@ -212,3 +241,86 @@ func checkLxcfsCfg() error {
 	}
 	return nil
 }
+
+// load daemon config file
+func loadDaemonFile(cfg *config.Config, flagSet *pflag.FlagSet) error {
+	configFile := cfg.ConfigFile
+	if configFile == "" {
+		return nil
+	}
+
+	contents, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
```

Comment:
If user passes a non-existent configfile, is it better to throw out the error rather than just ignore this?

Reply:
how about default config file is not exist? why default config file is not exist is corrent, and user-defined file is not exit is wrong.